src/mame/psikyo/psikyo.cpp

```diff
@@ -1161,7 +1156,7 @@ void psikyo_state::s1945(machine_config &config)
 	/* basic machine hardware */
 	M68EC020(config, m_maincpu, 16_MHz_XTAL);  // 16 MHz
 	m_maincpu->set_addrmap(AS_PROGRAM, &psikyo_state::s1945_map);
-	m_maincpu->set_vblank_int("screen", FUNC(psikyo_state::irq1_line_hold));
+	m_maincpu->set_vblank_int("screen", FUNC(psikyo_state::irq1_line_hold));  // Actually IPL2 being asserted (Level = 4).
```

Comment:
Can you correct it so it’s raising the correct interrupt level then?

Reply:
Done